.github/workflows/book.yml

```diff
@@ -7,6 +7,25 @@ on:
   merge_group:
 
 jobs:
+  up-to-date:
```

Comment:
It might not be the best place to add this CI job because it can't be canceled when a new commit is pushed and since it's a quite long-running job (because of the build part), it may be useful to be able to cancel in progress jobs. Feel free to move it where you want :)